taskfiles/tools.yml

```diff
@@ -10,17 +10,20 @@ tasks:
     desc: Install golangci-lint
     required:
       vars:
+        - TOOLS_BIN_DIR
```

Comment:
I think we could use `env` in this file to avoid both GOBIN and ad-hoc TOOLS_BIN_DIR var requirements